### PR TITLE
docs: add fallbck_sni faq

### PR DESCRIPTION
**Pull request by hf400159**
### Description
add fallbck_sni faq

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by SylviaBABY — DISMISSED** [review]

**Review by SylviaBABY — APPROVED** [review]
LGTM

File: docs/en/latest/FAQ.md [hunk 1]
```diff
@@ -281,6 +286,30 @@ To configure Apache APISIX to listen on multiple ports, you can:
 
 2. Reload or restart Apache APISIX.
 
+## After enabling the SSL certificate, why can't the corresponding route be accessed through HTTPS + IP?
```
Comment (tokers) [inline docs/en/latest/FAQ.md:289]: enabling? Maybe uploading?

**Review by SylviaBABY — APPROVED** [review]

File: docs/zh/latest/FAQ.md [hunk 2]
```diff
@@ -113,8 +119,8 @@ make deps ENV_LUAROCKS_SERVER=https://luarocks.cn
 
 举个例子，比如：`foo.com/product/index.html?id=204&page=2`，并考虑您需要根据查询字符串中的 `id` 在此条件下进行灰度发布：
 
-1. Group A：`id <= 1000`
-2. Group B：`id > 1000`
+1. Group A:`id <= 1000`
```
Comment (juzhiyuan) [inline docs/zh/latest/FAQ.md:122]: In Chinese document, I personally suggest using `：` instead of `:`

File: docs/zh/latest/FAQ.md [hunk 3]
```diff
@@ -113,8 +119,8 @@ make deps ENV_LUAROCKS_SERVER=https://luarocks.cn
...
+2. Group B:`id > 1000`
```
Comment (hf400159) [inline docs/zh/latest/FAQ.md:123]:
```suggestion
1. Group A：`id <= 1000`
2. Group B：`id > 1000`
```

File: docs/zh/latest/FAQ.md (hunk 2 above)
Comment (hf400159) [reply docs/zh/latest/FAQ.md]: fixed

**Review by SylviaBABY — APPROVED** [review]

File: docs/en/latest/FAQ.md [hunk 4]
```diff
@@ -281,6 +286,30 @@ To configure Apache APISIX to listen on multiple ports, you can:
 
 2. Reload or restart Apache APISIX.
 
+## After uploading the SSL certificate, why can't the corresponding route be accessed through HTTPS + IP?
+
+If you directly use HTTPS + IP address to access the server, the server will use the IP address to compare with the bound SNI. Since the SSL certificate is bound to the domain name, the corresponding resource cannot be found in the SNI, so that the certificate will be verified. The authentication fails, and the user cannot access the gateway via HTTPS + IP.
```
Comment (juzhiyuan) [inline docs/en/latest/FAQ.md:291]: image

emmm Two duplicated paragraphs?

File: docs/en/latest/FAQ.md [hunk 5]
```diff
@@ -281,6 +286,30 @@ To configure Apache APISIX to listen on multiple ports, you can:
...
+
+You can implement this function by setting the `fallback_sni` parameter in the configuration file and configuring the domain name. When the user uses HTTPS + IP to access the gateway, when the SNI is empty, it will fall back to the default SNI to achieve HTTPS + IP access to the gateway.
+
+```yaml title="./conf/config.yaml"
+apisix
+  ssl：
+    fallback_sni: "${your sni}"
+```
+
```
Comment (hf400159) [inline docs/en/latest/FAQ.md:312]:
```suggestion
```

File: docs/en/latest/FAQ.md (hunk 4 above)
Comment (hf400159) [reply docs/en/latest/FAQ.md:291]: fixed

**Review by juzhiyuan — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Pull request merged**